server/src/test/java/io/crate/integrationtests/TableSettingsTest.java

```diff
@@ -88,8 +88,8 @@ public void testSelectSettingsColumn() throws Exception {
 
     @Test
     public void testSetNonDynamicTableSetting() {
-        assertThrows(() -> execute("alter table settings_table set (\"translog.sync_interval\"='10s')"),
-                     isSQLError(containsString("Can't update non dynamic settings [[index.translog.sync_interval]] for open indices"),
+        assertThrows(() -> execute("alter table settings_table set (\"soft_deletes.enabled\"='true')"),
```

Comment:
Since `translog.sync_interval` is dynamic now we need to test against a different non dynamic setting.